src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.CSharp.targets

```diff
@@ -12,6 +12,7 @@ Copyright (c) .NET Foundation. All rights reserved.
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+    <AppDesignerFolder Condition="'$(AppDesignerFolder)' == ''">Properties</AppDesignerFolder>
```

Comment:
@davkean We're only publishing `None` if it has appropriate `CopyToPublishDirectory` metadata, which is the same as `Content`.  But I don't think that's the issue, we could always remove it from `Content` and add it to `None`, but we need to know which file to do it to.

The Web SDK wants to publish all *.json files under the project folder, except the launchSettings.json file.  So we need some way of specifying the file that should be an exception.  The `AppDesignerFolder` property is one way to do that, and another way to do it would be to exclude all files named launchSettings.json.  That's all I can think of.